src/main/resources/db/migration/V1__INIT_DB.sql

```diff
@@ -0,0 +1,92 @@
+-- MySQL Script generated by MySQL Workbench
```

Comment:
I guess there is SQL file in resource with the same content you need to delete as well